vars/githubWorkflowRun.txt

```diff
@@ -0,0 +1,116 @@
+Run workflow on github actions
+
+### Run as step:
+
+```
+  Map runInfo = githubWorkflowRun(repo: "owner/repository", workflow: "build.yml", ref: "main",
+    parameters: [path: "filebeat"], credentialsId: "github-workflow-token")
+```
+
+### Run asyncronous:
+
+```
+  script {
+    Map args = [
+       repo: "owner/repository",
+       workflow: "build.yml", 
+       ref: "main",
+       parameters: [path: "filebeat"],
+       credentialsId: "github-workflow-token"]
+    int runId = githubWorkflowRun.triggerGithubActionsWorkflow(args)
+    Map runInfo = githubWorkflowRun.getWorkflowRun(args + [runId: runId]) 
+  }
+
+```
+
+### Arguments:
+
+* workflow: workflow file name. Mandatory argument.
+* repo: repositiry owner and name. Optional, if not specifed used ifo from
+        ORG_NAME and REPO_NAME ennviroment variables.
+* ref: reference (branch, tag or hash). Optional, default is master.
+* parameters: map with parameters to pass to the workflow as inputs. Optional.
+* buildTimeLimit: How long wait till the run completed. It's set in hours, default is 1 hour.
+* credentialsId: github credetials id. Optional.
```

Comment:
This is not optional as far as I see it requires to be set otherwise it will be empty.

Reply:
Fixed.